gap/Utils.gd

```diff
@@ -27,7 +27,7 @@ DeclareGlobalFunction("GeneratorsOfOrthogonalGroup");
 #! @Description
 #! Return a matrix group <C>G</C> over the field <A>F</A> generated by the matrices from
 #! the list <A>gens</A>; the generators of <C>G</C> are immutable matrices over
-#! <A>F</A>. The attribute <Ref Func="DefaultFieldOfMatrixGroup"/> for the matrix
+#! <A>F</A>. The attribute <Ref Attr="DefaultFieldOfMatrixGroup"/> for the matrix
 #! group constructed will be <A>F</A>.
```

Comment:
This is wrong, even if `F` is finite: it (currently) only holds for fields with <= 256 elements, as then `ImmutableMatrix` will convert the matrices into "compressed" matrices which store the field explicitly. Indeed:
```
gap> F:=GF(257^2);
GF(257^2)
gap> G:=MatrixGroup(F, [IdentityMat(2,F)]);
Group([ [ [ Z(257)^0, 0*Z(257) ], [ 0*Z(257), Z(257)^0 ] ] ])
gap> DefaultFieldOfMatrixGroup(G);
GF(257)
```

Reply:
OK, I see you changed the code since I wrote this, so I guess the comment above is moot...